### ISSUE TITLE
DatabaseTransactions testing trait doesn't work

### ISSUE THREAD

**Issue by shadoWalker89** [laravel/framework#13964]
Hi,

I have multiple tests that use DatabaseTransactions, when i wrote the tests i was using laravel framework 5.2.27 and everything worked fine.

Then i updated to 5.2.36 and some of my tests started failing. Then i found out that the rows created by the tests are not being rolled back.

The problem is in one of the versions between 5.2.27 and 5.2.36

**Comment by GrahamCampbell** [comment]
Please could you narrow it down to the exact version it broke.

**Comment by shadoWalker89** [comment]
Found the problem,
First of all this is not a laravel problem.

The problem is from wampserver 3.

I installed wampserver 3 + the mysql addon for version 5.6.31 and then updated laravel to 5.2.36, turns out that the default storage engine is set to MyISAM, which doesn't support database transactions.

May be on the laravel, "laravel/laravel" project, the default engine for mysql should be set to InnoDB to avoid this kind of problems.
